app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function($filter) {
```

Comment:
Instead of taking in `$filter`, use `annotationFilter` here. Then you can call it directly below.